### DISCUSSION TITLE
How can I remove some surface points before computing the model

### DISCUSSION THREAD

**Discussion by Anagabrielamantilla** [gempy-project/gempy#926] [Q&A]
Hi, I am doing a model in Gempy 2024 version and would like to know how can a specific surface point be removed before computing the model. For example:

```
gp.add_surface_points(
    geo_model=modelo,
    x=[500],
    y=[500],
    z=[-850],
    elements_names=['surface2']
)

```
If I add some points to a model with this function, but now I do not want to use some in computing the interpolation, how can I remove it? I would be very thankful for any help.

**Comment by javoha** [comment]
Hi and sorry for the late reply,
I don't think there is a function to remove single surface points at the moment. The easiest way is to directly make changes on the input file (e.g. using pandas). 
What exactly are you trying to do, maybe there are better alternatives depending on your workflow.
Cheers,
Jan
